BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -1,5 +1,18 @@
 {% extends 'BraincraftedBootstrapBundle:Form:bootstrap.html.twig' %}
 
+{% block form %}
```

Comment:
I'm not a big fan of the flashbag in this context. But can't we use ``{{ form_error(form) }}``